packages/integration-sdk-runtime/src/execution/dependencyGraph.ts

```diff
@@ -267,7 +270,7 @@ export function executeStepDependencyGraph<
                   stepId,
                   cached: hasCachePath(stepId).toString(),
                 },
-                operation: () => executeStep(step),
+                operation: () => executeStep(step, stepWrapper),
```

Comment:
Just curious, I see you decided to pass stepWrapper into `executeStep` and then only wrap part of the function (where we call `step.executionHandler`. What's the difference if we instead just wrap all of `executeStep`? Is there any difference in results in terms of spans? Then we could do something like below.

```typescript
let operation: () => Promise<void>;
if (stepWrapper) operation = () => { return stepWrapper(step, executeStep(step)); };
else operation = () => executeStep(step);
```

Reply:
Because the error is caught on line 386, we never get the error shown in the trace